src/main/java/at/hannibal2/skyhanni/utils/SkyBlockItemModifierUtils.kt

```diff
@@ -121,6 +121,34 @@ object SkyBlockItemModifierUtils {
         return null
     }
 
+    fun ItemStack.getHelmetSkin(): String? {
+        for (tags in tagCompound.keySet) {
+            if (tags != "ExtraAttributes") continue
+            val extraAttributes = tagCompound.getCompoundTag(tags)
+            for (attributes in extraAttributes.keySet) {
+                if (attributes == "skin") {
+                    return extraAttributes.getString(attributes)
+                }
+            }
+        }
+
+        return null
```

Comment:
```suggestion
        return tagCompound?.getCompoundTag("ExtraAttributes")?.getString("skin")?.takeUnless { it.isBlank() }
```
This should be fine, since `getTagCompound` returns an empty default tag if that key is not present (similar with `getString` returning a blank string by default)